tests/test_qubit_device.py

```diff
@@ -583,6 +581,10 @@ class TestMarginalProb:
             ([0, 2], [1]),
             ([1, 2], [0]),
             ([0, 1, 2], []),
```

Comment:
Just adding some examples with ``Wires`` input to check if the function takes them.